BackEndApp/authv1/store.py

```diff
@@ -32,7 +32,11 @@ def delete(self, project=None):
             _path = self.path
             if project is not None:
                 _path = posixpath.join(_path, project)
-            shutil.rmtree(_path)
+            if project.startswith("shared_"):
+                os.remove(_path)
+            else:
+                shutil.rmtree(_path)
```

Comment:
```suggestion
            os.remove(_path)
```

Reply:
Oww, you can look at this suggestion or let me know if it won't work @therajtiwari

Reply:
It actually won't work. Can you please check my Slack message?